docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,542 @@
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "activations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee505c2f-e048-4b2d-824d-d9e6cd2f934f",
+   "metadata": {},
+   "source": [
+    "This object is iterable:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "40e2bac8-2dfb-4633-a888-c2a80f7c6278",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "job-failure-alert\n",
+      "job-failure-alert\n",
+      "job-failure-alert\n",
+      "job-failure-alert\n"
+     ]
+    }
+   ],
+   "source": [
+    "for activation in activations:\n",
+    "    print(activation.name)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "2c8d2fa5-3d45-49c6-b96e-7666b4465ed4",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "> 2024-12-11 11:52:10,062 [info] Project loaded successfully: {\"project_name\":\"default\"}\n"
+     ]
+    }
+   ],
+   "source": [
+    "project = mlrun.get_or_create_project(\"default\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "a012c3d0-11ce-41f1-bc9b-e8fb8d4967b3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# list alert activations for a specific project\n",
+    "activations = project.list_alert_activations(\n",
+    "    name=None,\n",
+    "    since=None,\n",
+    "    until=None,\n",
+    "    entity=None,\n",
+    "    severity=None,\n",
+    "    entity_kind=None,\n",
+    "    event_kind=None,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "5ae0441c-5105-47c3-aa43-78ce8215f8fd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# list alert activations for a specific alert config\n",
+    "alert_config = run_db.list_alerts_configs()[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "842fa453-a434-4794-a1a4-149b65d900d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "activations = alert_config.list_activations(\n",
+    "    since=None,\n",
+    "    until=None,\n",
+    "    from_last_update=False,  # set to True to get activations only from the time when alert config was updated (takes precedence over since of both are passed)\n",
```

Comment:
" from_last_update=False, # set to True to get activations only from the time when alert config was updated (takes precedence over "since" if both are passed)\n",